Automattic/jetpack
# Blocks: update deprecated block editor selector

Fixes #28719

## Proposed changes:

See #27795

As a deprecated selector will get removed in 6.2, it's time for us to update.

### Other information:


## Jetpack product discussion

* N/A

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Check out this branch.
* Activate the Publicize feature under Jetpack > Settings > Sharing
* Connect a Twitter account
* Go to Posts > Add New
* Add the following at the end of the URL and refresh the page: `?anchor_podcast=22b6608&anchor_episode=e324a06c-3148-43a4-85d8-34c0d8222138&spotify_show_url=https://open.spotify.com/show/6HTZdaDHjqXKDE4acYffoD?si=EVfDYETjQCu7pasVG5D73Q`
* When the page is refreshed, you should see new content being added.
* Add some more content manually.
* Go to the Jetpack sidebar in the editor, and switch the Twitter options to share a Twitter Thread instead of a single tweet.
* As you continue to edit the post content, you should see indicators in the post content, showing you where it will be divided into multiple tweets.
